### ISSUE TITLE
Permission denied (publickey).

### ISSUE THREAD

**Issue by agersoncgps** [GeekyAnts/native-base-cli#1]
```
$ npm install -g native-base-cli
/usr/local/bin/native-base -> /usr/local/lib/node_modules/native-base-cli/index.js
/usr/local/lib
└─┬ native-base-cli@1.0.0 
  └─┬ fs-extra@3.0.1 
    ├── graceful-fs@4.1.11 
    ├── jsonfile@3.0.1 
    └── universalify@0.1.1 
```

```
$ native-base init
Cloning into 'native-base-boilerplate'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by sankhadeeproy007** [comment]
@agersoncgps Hi are you sure you've added your ssh key to github?

**Comment by digitalsleep** [comment]
can i set this to use bitbucket instead?

**Comment by sankhadeeproy007** [comment]
@digitalsleep You can port your project to bitbucket later. But you'll need github to clone the project initially.

**Comment by grigy** [comment]
I'm also running into this issue. Is there any workaround?

**Comment by nandesu** [comment]
Yes. Add your public SSH key to your github account.
https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/

**Comment by SupriyaKalghatgi** [comment]
Yes adding your ssh key to your github account is the only solution for this

**Comment by grigy** [comment]
@nandesu, @SupriyaKalghatgi thank you!

**Comment by SupriyaKalghatgi** [comment]
@agersoncgps @grigy Is the issue solved?

**Comment by reza7rm** [comment]
I have added my public ssh key to my account still get Permission denied error!

**Comment by SupriyaKalghatgi** [comment]
@reza7rm Thats something weird!
But adding ssh key is the solution for this
If you have any github projects cloned on your system, then it should be the same for this also

**Comment by grigy** [comment]
@agersoncgps yes! and sorry for late answer

**Comment by jkltech** [comment]
I am getting 

"fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists."

Even after adding the SSH Keys to my account

Should I get permission to access the repositories of nativebase??

Fixed: Ensure that your id_rsa private key file is having a 600 permission (no read write permission for group and others). And also have your passphrase of private key handy.

**Comment by agliottone** [comment]
i have resolved with this guide
https://support.automaticsync.com/hc/en-us/articles/202357115-Generating-an-SSH-Key-on-Windows

**Comment by misuzu** [comment]
@sankhadeeproy007 Why not just use ```https://github.com/GeekyAnts/native-base-boilerplate.git``` instead of ```git@github.com:GeekyAnts/native-base-boilerplate.git```?
